File: 백준/Bronze/2566. 최댓값/최댓값.py
mtx = []
for i in range(9):
    line = list(map(int, input().split()))
    mtx.append(line)

max_line = 0
max_num = 0
for i in mtx:
    if max_num < max(i): 
        max_num = max(i)
        max_line = mtx.index(i)

print(max_num)
print(max_line+1, mtx[max_line].index(max_num)+1)

File: 백준/Bronze/2566. 최댓값/최댓값.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	r := bufio.NewReader(os.Stdin)
	w := bufio.NewWriter(os.Stdout)

	mtxA := make([][]int, 9)

	for i := 0; i < 9; i++ {
		mtxA[i] = make([]int, 9)
	}

	for i := 0; i < 9; i++ {
		strInput, _ := r.ReadString('\n')
		row := strings.Split(strings.TrimSpace(strInput), " ")
		for j, rowStr := range row {
			val, _ := strconv.Atoi(rowStr)
			mtxA[i][j] = val
		}
	}

	maxLine := 0
	maxNum := 0
	maxIndex := 0

	for i, m := range mtxA {
		for j, s := range m {
			if maxNum < s {
				maxNum = s
				maxLine = i
				maxIndex = j
			}
		}
	}

	fmt.Fprintln(w, maxNum)
	fmt.Fprintln(w, maxLine+1, maxIndex+1)

	w.Flush()
}
